src/ngraph/distributed/mlsl.hpp

```diff
@@ -83,10 +83,23 @@ namespace ngraph
                     throw std::runtime_error("AllReduce op supports only f32 and f64 types");
                 }
 
+                auto allReduce_type = MLSL::RT_SUM;
```

Comment:
We generally do not use `camelCase` in nGraph. (Well, just in the name of the product, I guess. :))